clang-format.sh

```diff
@@ -14,6 +15,4 @@ else
     filter="grep ."
 fi
 
-find */ -type f -name \*.[ch] -o -name \*.[ch]pp -o -name *.cc | \
-    $filter | \
-    xargs clang-format -i -style=file
+git ls-files | grep -E '.*\.([ch](pp)?|cc)$' | $filter | xargs clang-format -i -style=file
```

Comment:
git ls-files takes a `files...` argument which is the same glob-like pattern that is used in gitignore. It would be better to use that here. (nitpick)

Reply:
Awesome! Thanks for the suggestion.
The man-page doesn't make it clear what <file...> is supposed to be (it does only for the exclude patterns) but seems to work the way you explain it, so I'll use it.